.gitignore

```diff
@@ -1,54 +1 @@
-# .gitignore defines files to ignore and remain untracked in Git
-# Each line in a gitignore file specifies a pattern, eg. directory or file extension
-
-# Git should not track binary artifacts such as images, libraries, executables, archive files etc.
-# Until the team has mature processes, a Binary Artifacts Repository Manager is not in use.
-
-# Therefore some binary artifacts are tracked in this Git repository
-
-
-# Further .gitignore templates available at:
-# https://github.com/github/gitignore
-
-
-# macOS OS generated files
-.DS_Store
-._*
-.Spotlight-V100
-.Trashes
-
-# Windows OS generated files #
-ehthumbs.db
-Thumbs.db
-
-# Compressed Archives
-# git has built-in compression
-# *.7z
-# *.dmg
-# *.gz
-# *.iso
-# *.jar
-# *.rar
-# *.tar
-# *.zip
-
-# Binaries / Compiled source
-# *.com
-# *.class
-# *.dll
-# *.exe
-# *.o
-# *.so
-
-# Logs and databases
-# *.log
-# *.sqlite
-
-# Byte-compiled / optimized / DLL files
-__pycache__/
-*.py[cod]
-*$py.class
-
-# VSCode
-.vscode
-
+plugins/modules/user.py
```

Comment:
gitignore file replaced by one line seems like a mistake. We should keep this generic gitignore to avoid commits including OS generated files and other accidents like pycache that will bloat the git repo commit history

Reply:
yes, this is a mistake